stylus/components/forms.styl

```diff
@@ -311,13 +311,13 @@ $checkbox
             box-shadow inset 0 0 0 rem(2) dodgerBlue
 
         &::after
-            background-image embedurl('../../assets/icons/ui/check.svg')
+            background-image embedurl('../../assets/icons/ui/check-white.svg')
```

Comment:
Colorful icon again, we may need to put them on a separate icon folder one day to avoid use them with `<Icon />`

Reply:
Yup.
But isn't your incoming work on the SVG sprite specifically target a list of icon? It doesn't take them all, does it?